packages/vaadin-themable-mixin/register-styles.js

```diff
@@ -4,3 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 export { registerStyles, css, unsafeCSS } from './vaadin-themable-mixin.js';
+
+export const addGlobalThemeStyles = (id, prefix, ...styles) => {
```

Comment:
Should this be in `vaadin-themable-mixin.js` and re-exported here (like `registerStyles`)?

Reply:
I put it here because it is not something that "ordinary" users would ever need. I can move it if you feel it should be moved

Reply:
No need, if it's mainly for internal purposes. Could be documented so.

Reply:
Added docs